README.md

```diff
@@ -1,20 +1,25 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Capstone Two - API manipulation & JavaScript project
-Simple recipe menu display using real API calls!
+![view](https://user-images.githubusercontent.com/77069011/169627999-45be81b3-d4bb-47bc-aa04-78bc63584e8c.jpeg)
 
+# Capstone Two - Yummy Recipes - API manipulation & JavaScript project
+
+Recipe webpage that has many meal categories and a lot of options for each one of them. Importing data from a real API and implementing an involvement system that allows the user to like each recipe and to add comments to them.
 
 ## Built With
+
 - HTML, JavaScript, CSS
 - Webpack
 - Gitflow
 - TheMealDB API
+- Involvement API (Microverse's)
 
 ## Live Demo
-- [Demo](https://anagudelogu.github.io/capstone-two/dist/)
 
+- [Demo](https://anagudelogu.github.io/capstone-two/dist/)
 
 ## Getting Started
```

Comment:
[OPTIONAL] Great job documenting your project! It would be great to add some instructions for running the app like `npm install` and other basic commands, this way anyone will be able to run it.